text_classifier/sigopt_creds.py

```diff
@@ -0,0 +1,4 @@
+# Find your creds at https://sigopt.com/user/profile
```

Comment:
That would actually be pretty slick I think.  Would it be quick to add?  You could just generate the file automatically I suppose